### DISCUSSION TITLE
[✨] Named slots based on element type

### DISCUSSION THREAD

**Discussion by DustinJSilk** [QwikDev/qwik-evolution#173] [Proposals For Qwik]
### Is your feature request related to a problem?

The process for using named Slots can be quite verbose. Every element being projected into a named slot requires `q:slot=“…”`, even when there may already be identifiable properties on the elements (ie element type, ID, class, attribute)  

### Describe the solution you'd like

Add new props to the Slot component that allows you to project content based on the element type.

For example:

`<Slot element=“p” />`

This would only project `<p />` elements, similar to the Slots name property.

This behaviour is similar to how Angulars ng-content allows for matching based on selectors.

### Describe alternatives you've considered

Simply using named slots works but doesn’t have great DX.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
